### PR TITLE
contact form

### PR DESCRIPTION
**Pull request by uno36**
Added images
updated html and css

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, where linter checks are not running. Kindly fix the conflict so linter checks can run.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,252 +1,252 @@
...
-        
\ No newline at end of file
+                    <li><a href="#"><img src="images/ic_stackoverflow.png" alt=""></a><li>
+                    <li><a href="#"><img src="images/githubicon.png" alt=""></a><li>
+                    <li><a href="#"><img src="images/twittericon.png" alt=""></a><li>
+                    <li><a href="#"><img src="images/linkedin.png" alt=""></a><li>
+                    <li><a href="#"><img src="images/medium (2).png" alt=""></a><li>
+                    <li><a href="#"><img src="images/angellist (2).png" alt=""></a>
+                    <li>
+                </ul>
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] So far almost everything is working well 👏 👍 . However, we need to ensure that the portfolio is professional. If you could have met some of the articles, the traditional horizontal scrollbar burdens the user by requiring constant attention and greater physical effort to maintain the dragging 😞 . Therefore kindly ensure that there is no horizontal scroll effect ☺️ .

Your design 😎 
image

File: index.html [hunk 2]
```diff
@@ -1,252 +1,252 @@
...
-                <div  class="get-startedcollaboration">
-                <p class="get-started-start-collab">start collaboration <p>
-                </div>
-                <div class="contact-img">
-                    <button class="resume-btn"><a href="#"> Get my Resume  <img class="share-icon" src="images/icons8-share-24.png" alt="union"></a></button>
-                </div>
+                <form action="https://formspree.io/f/xgebyodb" method="post">
+                    <input type="text" class="name" name="name" placeholder="your name" required="true" maxlength="30" > <br>
+                    <input type="email" class="email" name="Email" required="true" placeholder="your email">
+                    <textarea name="text-area" id="your-message" rows="5" class="message" required="true" placeholder="Enter your message..."></textarea>
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) suggest, make sure that the message field accepts up to 500 chars ☺️ . This is to ensure the user is limited to writing a wrong message that may burden you to read 📖 .

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @uno36 🙋‍♂️
Good job working on your first contact form. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶 : 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
- [ ] [OPTIONAL]  Consider the use of [`<fieldset>`](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form#the_fieldset_and_legend_elements)
If you notice for form [accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility) reasons wrapping the inputs in the `<fieldset>` is more semantic. Kindly make the form more accessible to screen readers such us [Jaws](https://www.freedomscientific.com/products/software/jaws/) and [NVDA](https://www.nvaccess.org) by using `<fieldset>`.



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @uno36 , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.
💯 Amazing UI


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

**1-** As requested by the previous code reviewer Please consider removing the horizontal scroll from your web page to make it more use friendly and in accordance with the design template.

## Solution:-

Adding the following CSS in your design can remove the horizontal scroll bar in your case.

```

html {
    overflow-x: hidden;
}

```

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -2,6 +2,10 @@
 @import url('https://fonts.googleapis.com/css?family=Playfair Display');
 @import url('https://fonts.googleapis.com/css?family=Poppin');
 
+html {
+  overflow-x: hidden;
```
Comment (souad988) [inline styles.css:6]: Good job on trying to fix the horizontal scrolling issue , To solve this problem move this property to the body

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @uno36 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

212385281-4797c58a-d358-423a-aa7c-832c22203865


 ## Highlights
✔Very well implemented design 🥳💪
✔Use correct github workflow
✔No linter issues
✔Overall work is good  

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by souad988 — APPROVED** [review]
Hi @uno36 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6a4a5143327075565a70544d4f34765573302f67697068792e676966


## Highlights
✔Good design 
✔Good use of github workflow
✔Good PR title

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**